scripts/double_links.py

```diff
@@ -119,13 +119,59 @@ def delete_links_and_output_results(self):
         print "Midrash links: {}".format(self.midrash)
 
 
+
+    def get_exact_doubles(self):
+        ls = LinkSet()
+        link_refs = {}
+        count = 0
+        for l in ls:
+            refs = tuple(sorted(l.refs))
+            if refs in link_refs:
+                count += 1
+                if refs not in self.bad_ids:
+                    prev = link_refs[refs]
+                    self.bad_ids[refs] = [l._id, prev]
```

Comment:
Some links are more preferable than others.  If it has `source_text_oid`, it's preferable.  If not that, presence of `auto` or `generated_by` makes them preferable.